### DISCUSSION TITLE
Need help for AdminInADay training here in GitHub

### DISCUSSION THREAD

**Discussion by WanMKI** [microsoft/powerapps-tools#1187] [Q&A]
I am trying to do training AdminInADay however during setting up the test environment, I am having issue with Step 2 when running the command Start-AdminInADay-Setup . I am having below error as attached.
![MicrosoftTeams-image (15)](https://user-images.githubusercontent.com/116694680/197913799-70920ca3-cf4d-4de2-ae80-72cbe43fc705.png)

Does the command require local admin rights to our laptop? I am doing from my work laptop which have no admin rights.

**Comment by NL12143** [comment]
What repository is this coming from ? 
First look you did not install, import, the modules.  Yes, you can run these commandlets as a user. But admin is better. 
Install-Module -Name Microsoft.Online.SharePoint.PowerShell -Scope CurrentUser
